include/mbedtls/psa_util.h

```diff
@@ -93,6 +95,7 @@ static inline psa_algorithm_t mbedtls_psa_translate_cipher_mode(
             if( taglen == 0 )
                 return( PSA_ALG_CBC_NO_PADDING );
             /* Intentional fallthrough for taglen != 0 */
+            MBEDTLS_FALLTHROUGH;
             /* fallthrough */
```

Comment:
We're adding a macro to the public API. This macro is not fully portable. And I just realized that all this is to be used only twice, and just to avoid duplicating `return 0;`? The code would be shorter if we duplicated `return 0`! Let's do that instead.